spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Page Index', type: :feature do
+  before :each do
+    @user = User.create(
+      name: 'Tommy',
+      photo: 'https://images.unsplash.com/photo-1545167622-3a6ac756afa4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=712&q=80',
+      bio: 'Teacher',
+      posts_counter: 1
+    )
+    @post = Post.create(title: 'Hello',
+                        text: 'My world',
+                        comments_counter: 0,
+                        likes_counter: 0,
+                        author_id: @user.id)
+    5.times do
+      Comment.create(post: @post,
+                     author: @user,
+                     text: 'hello yemi',
+                     post_id: @post.id,
+                     author_id: @user.id)
+    end
+    visit user_posts_path(@user.id)
+  end
+
+  it 'shows the username of user' do
+    expect(page).to have_content(@user.name)
+  end
+
+  it 'shows number of posts of user' do
+    expect(page.body).to have_content(@user.posts_counter.to_s)
+  end
+
+  it 'shows the body of a post' do
+    expect(page).to have_content('My world')
+  end
+
+  it 'shows first comments on a post' do
+    expect(page).to have_content('hello yemi')
+  end
+
+  it 'shows section for pagination' do
+    expect(page).to have_content('Pagination')
+  end
+end
```

Comment:
- [ ] In the User post index page, you are expected to have 10 tests, but at the moment you are having just 5. Kindly check the attached screenshot and add all the missing tests
![Screen Shot 2023-02-15 at 3 30 18 AM](https://user-images.githubusercontent.com/62849872/219015823-4108bac6-3610-4619-81d7-8b6f4a34468c.png)